packages/analytics/dashboard-renderer/src/components/TopNTableRenderer.vue

```diff
@@ -10,15 +10,44 @@
       :description="chartOptions.description"
       :synthetics-data-key="chartOptions.syntheticsDataKey"
       :title="chartOptions.chartTitle || ''"
-    />
+    >
+      <template
+        v-if="props.chartOptions?.entityLink"
```

Comment:
Hm.  You shouldn't need conditional chains on `chartOptions` -- it's a required prop.  Did you run into trouble?